src/_assets/css/main.scss

```diff
@@ -357,6 +357,33 @@ img {
   overflow-x: hidden;
   overflow-y: auto;
   z-index: 1;
+
+  .site-sidebar {
+    .nav-link {
+      font-family: $site-font-family-base;
+      font-weight: light;
+      color: #4a4a4a;
+      font-size: 13px;
+      line-height: 26px;
+
+      &[data-toggle] {
+        &:not(.collapsable) {
+          font-family: $site-font-family-alt;
+          font-weight: regular;
+          font-size: 20px;
+          line-height: 25px;
```

Comment:
Delete line-height? (mockup has it as "auto", which resolves to 26px in my staged version)